### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by aesteve-rh**
Signed-off-by: Albert Esteve <aesteve@redhat.com>

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]

**Review by barpavel — COMMENTED** [review]
@nirs For your attention

File: README.adoc [hunk 1]
```diff
@@ -143,17 +143,30 @@ Both packages can be installed from ovirt-master-snapshot-static repository:
...
 
-  $ sudo yum install -y $(cat automation/check-patch.packages)
+- Enable oVirt packages for Fedora
++
+  $ sudo dnf copr enable -y nsoffer/ioprocess-preview
+  $ sudo dnf copr enable -y nsoffer/ovirt-imageio-preview
+
+- Install additional packages
++
+  $ sudo dnf install -y $(cat automation/check-patch.packages)
```
[the remark this answers is not in the document]
Comment (nirs) [reply README.adoc:169]: ioprocess is not relevant for engine host.

[the remark this answers is not in the document]
Comment (barpavel) [reply README.adoc:169]: @ahadas @nirs @bennyz 
I think the whole Fedora parts can be removed and some other parts updated.
The instructions are about Fedora 30 (!!!)
There is no `automation/check-patch.packages`.
The https://resources.ovirt.org/pub/ovirt-master-snapshot-static/rpm/ doesn't have a Fedora `fc` folder at all! Not it has an `el` folder, but `el8` & `el9` instead.
I canl update it :)

[the remark this answers is not in the document]
Comment (bennyz) [reply README.adoc:169]: @aesteve-rh please remove it in https://github.com/oVirt/ovirt-engine/pull/262

[the remark this answers is not in the document]
Comment (barpavel) [reply README.adoc:169]: @bennyz Maybe better have a separate "Fedora removal" PR, rather then doing part of the work in the #262 PR, that started as Markdown --> AsciDoc link fixes?

[the remark this answers is not in the document]
Comment (aesteve-rh) [reply README.adoc:169]: I just pushed it as you were commenting this, but I can still separate that commit into another PR if you think that's better. Let's move the dicussion to #262

[the remark this answers is not in the document]
Comment (bennyz) [reply README.adoc:169]: No need to separate

**Pull request merged**